### DISCUSSION TITLE
Specifying Keypress

### DISCUSSION THREAD

**Discussion by LCagmac** [DigiScore/neoscore#81] [Q&A]
Hi Neoscore,

I have been testing using key presses for a graphic score. I can get key presses to work, but I am having trouble setting my script up so actions only respond to certain key presses. Here is a picture of my code. What should I change in order to use only specific key presses?

Thank you!

<img width="785" alt="Screen Shot 2022-12-09 at 9 53 16 PM" src="https://user-images.githubusercontent.com/42069097/206825262-1945ab24-53e1-4417-ab34-6b4660d47ec8.png">

**Comment by ajyoon** [comment]
What's the behavior you're trying to get? vs what happens when you run this code?

You probably don't want to call `neoscore.show()` inside the key handler. The key handler only attaches and runs after `neoscore.show` was already called, and you can't (or at least shouldn't) have multiple global Qt applications running at the same time in one script.

**Reply by LCagmac** [reply]
I will test out reformating the code. 

Thank you!

With this, I was trying to use certain keycodes to start certain movements of the graphic score elements.

**Reply by ajyoon** [reply]
If you haven't already seen it, be sure to check out the [input-driven scene interaction example](https://github.com/DigiScore/neoscore/blob/main/examples/input_scene_interaction.py)

**Reply by LCagmac** [reply]
Thank you!
